### PR TITLE
[FAB-17818] Update documetation in Fabric samples repo

### PR DESCRIPTION
**Pull request by nikhil550**
Update READMEs to improve documentation in the samples repo.

Most importantly, for download instructions, we should use the topics in the Fabric docs to capture the most recent changes. I also think users would benefit from tables that would provide more detail on each sample and include a link to the accompanying tutorial. Some of these samples and chaincode, interest rate samples = state based endorsement, abac = attribute based access control, could help users find and take advantage of these samples.

Signed-off-by: NIKHIL E GUPTA <negupta@us.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,27 @@
...
-# Fetch bootstrap.sh from fabric repository using
-curl -sS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh -o ./scripts/bootstrap.sh
-# Change file mode to executable
-chmod +x ./scripts/bootstrap.sh
-# Download binaries and docker images
-./scripts/bootstrap.sh [version] [ca version] [thirdparty_version]
-```
+# Hyperledger Fabric Samples
+
+You can use Fabric samples to get started working with Hyperledger Fabric, explore important Fabric features, and learn how to build applications that can interact blockchain networks using the Fabric SDK's. To learn more about Hyperledger Fabric, visit the [Fabric documentation](https://hyperledger-fabric.readthedocs.io/en/master).
```
Comment (lehors) [inline README.md:5]: interact -> interact with
SDK's -> SDKs

File: README.md [hunk 2]
```diff
@@ -1,33 +1,27 @@
...
+
+## Getting started with the Fabric samples
+
+To use the Fabric samples, you need to download the Fabric Docker images and the Fabric CLI tools. First, make sure that you have installed all of the [Fabric prerequisites](https://hyperledger-fabric.readthedocs.io/en/master/prereqs.html). You can then follow the instructions to [Install the Fabric Samples, Binaries, and Docker Images](https://hyperledger-fabric.readthedocs.io/en/master/install.html) in the Fabric documentation. In addition to downloading the Fabric images and tool binaries, the instructions will clone the Fabric samples on your local machine.
```
Comment (lehors) [inline README.md:9]: the instructions will clone -> the instructions will make you clone

File: README.md [hunk 3]
```diff
@@ -1,33 +1,27 @@
...
+
+## Guide to the Fabric samples
+
+You can use the following table to learn more about each sample, and find the corresponding tutorial or documentation.
+
+|  **Sample** | **Description** | **Documentation** |
+| -------------|------------------------------|------------------|
+| [Fabric test network](test-network) | Get started by deploying a basic Fabric network on your local machine. | [Using the Fabric test network](https://hyperledger-fabric.readthedocs.io/en/master/test_network.html) |
+| [Fabcar](fabcar) | Lean how to use the Fabric SDK's to invoke smart contracts from your client applications. | [Writing your first application](https://hyperledger-fabric.readthedocs.io/en/master/write_first_app.html) |
```
Comment (lehors) [inline README.md:18]: Lean -> Learn

File: README.md [hunk 4]
```diff
@@ -1,33 +1,27 @@
...
+| [Commercial paper](commercial-paper) | Explore a use case in which two organizations use a blockchain network to trade commercial paper. | [Commercial paper tutorial](https://hyperledger-fabric.readthedocs.io/en/master/tutorial/commercial_paper.html) |
+| [Interest rate swaps](interest_rate_swaps) | Explore state based endorsement using a financial services use case. | [Setting Key level endorsement policies](https://hyperledger-fabric.readthedocs.io/en/master/endorsement-policies.html#setting-key-level-endorsement-policies) |
+| [Off chain data](off_chain_data) | Learn how to use the Peer channel-based event services to build an off chain database for reporting and analytics. | [Peer channel-based event services](https://hyperledger-fabric.readthedocs.io/en/master/peer_event_services.html) |
+| [High throughput](high-throughput) | Learn how you can design your smart contracts to process a large volume of transactions. | |
+| [First network](first-network) | **Deprecated** | [Build your first network](https://hyperledger-fabric.readthedocs.io/en/master/build_network.html) |
```
Comment (lehors) [inline README.md:23]: I think it's worth adding something like "Deprecated in favor of Fabric test network."

File: README.md [hunk 5]
```diff
@@ -1,33 +1,27 @@
...
+| [Chaincode](chaincode) | A set of sample smart contracts used by other samples and the tutorials in the Fabric documentation | [Fabric tutorials](https://hyperledger-fabric.readthedocs.io/en/master/tutorials.html) |
```
Comment (lehors) [inline README.md:24]: You missing the period at the end. :-)

Comment (nikhil550) [reply README.md:24]: done.

File: README.md (hunk 3 above)
Comment (nikhil550) [reply README.md:18]: done.

File: README.md (hunk 2 above)
Comment (nikhil550) [reply README.md:9]: done

File: README.md (hunk 1 above)
Comment (nikhil550) [reply README.md:5]: done

**Pull request merged**